src/main/java/com/blackducksoftware/integration/jira/web/action/MigrationActions.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Black Duck JIRA Plugin
+ *
+ * Copyright (C) 2020 Synopsys, Inc.
+ * https://www.synopsys.com/
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.blackducksoftware.integration.jira.web.action;
+
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.TimeZone;
+import java.util.stream.Collectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.atlassian.scheduler.SchedulerService;
+import com.atlassian.scheduler.SchedulerServiceException;
+import com.atlassian.scheduler.config.JobConfig;
+import com.atlassian.scheduler.config.JobId;
+import com.atlassian.scheduler.config.JobRunnerKey;
+import com.atlassian.scheduler.config.RunMode;
+import com.atlassian.scheduler.config.Schedule;
+import com.blackducksoftware.integration.jira.data.accessor.JiraSettingsAccessor;
+import com.blackducksoftware.integration.jira.data.accessor.MigrationAccessor;
+import com.blackducksoftware.integration.jira.issue.model.ProjectMappingConfigModel;
+import com.blackducksoftware.integration.jira.task.maintenance.AlertMigrationRunner;
+import com.blackducksoftware.integration.jira.web.model.BlackDuckJiraConfigSerializable;
+import com.blackducksoftware.integration.jira.web.model.JiraProject;
+import com.blackducksoftware.integration.jira.web.model.MigrationDetails;
+
+public class MigrationActions {
+    private static final String MIGRATION_JOB_NAME = MigrationActions.class.getName() + ":migration";
```

Comment:
would getClass() work better here than the explicit MigrationActions.class reference? This way you don't have to change this value if the class name changes.

Reply:
getClass() is not a static method that I can use to create this static String